sycl/test/built-ins/vector_relational.cpp

```diff
@@ -570,5 +571,35 @@ int main() {
     assert(r4 == 34.34f);
   }
 
+  {
+    s::vec<int, 4> r(0);
+    {
+      s::vec<int, 4> a(1, 2, 3, 4);
+      s::vec<int, 4> b(5, 6, 7, 8);
+      s::vec<unsigned int, 4> m(1u, 0x80000000u, 42u, 0x80001000u);
+      s::buffer<s::vec<int, 4>> A(&a, s::range<1>(1));
+      s::buffer<s::vec<int, 4>> B(&b, s::range<1>(1));
+      s::buffer<s::vec<unsigned int, 4>> M(&m, s::range<1>(1));
+      s::buffer<s::vec<int, 4>> R(&r, s::range<1>(1));
+      s::queue myQueue;
+      myQueue.submit([&](s::handler &cgh) {
+        auto accA = A.get_access<s::access::mode::read>(cgh);
```

Comment:
Minor nit to match previous tests.
```suggestion
        auto AccA = A.get_access<s::access::mode::read>(cgh);
```

Reply:
Done